src/Evolution/Initialization/Domain.hpp

```diff
@@ -91,7 +91,7 @@ struct Domain {
     ElementMap<Dim, Frame::Inertial> map{element_id,
                                          my_block.coordinate_map().get_clone()};
 
-    return std::make_tuple(add_nonexistent_to_databox<Domain, compute_tags>(
+    return std::make_tuple(merge_into_databox<Domain, compute_tags>(
         std::move(box),
         tuples::tagged_tuple_from_typelist<simple_tags>{
             std::move(mesh), std::move(element), std::move(map)}));
```

Comment:
I think it would be cleaner to skip the conversion to the tagged tuple and just pass the simple tags values as arguments, as is done in `db::create`

Reply:
Can't be done in a safe manner. The only other option might be `merge_into_databox<Domain, simple_tags, compute_tags>(std::move(box), std::move(BLAH)...)`

Reply:
I like it because it's what `db::create` does

Reply:
I like it because it's what `db::create` does